OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -155,14 +155,15 @@ public void UpdateVisibility()
 			// PERF: Avoid LINQ.
 			foreach (var puv in Footprint)
 			{
-				if (shroud.IsVisible(puv))
+				var cv = shroud.Visibility(puv);
+				if ((cv & Shroud.CellVisibility.Visible) != 0)
```

Comment:
We should use the hasFlag method, ex: `cv.HasFlag(Shroud.CellVisibility.Visible)`